composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.6.0",
-        "robmorgan/phinx": "^0.10.3|^0.11.1",
+        "robmorgan/phinx": "^0.10.3|^0.11.1|^0.12.1",
```

Comment:
Won't 0.12 also require CakePHP4 which will conflict with a 3.x application?

Reply:
Maybe thats why it worked for me, since it silently didn't pull it. You are right, closing then.

Reply:
That would work, though, if 
- [ ] https://github.com/cakephp/phinx/pull/1753
- [ ] https://github.com/cakephp/phinx/issues/1754

would be applied (not sure if we should).